source/hardware/controllers/denon_mc7000.rst

```diff
@@ -298,27 +304,28 @@ Layout and functions
     the current loop. Press and hold Shift and then press this button to
     create a Loop Out point at the current Location.
 
-43. **< / > Param 1/2:** These are currently used to beat jump backward
-    or forward by 8 beats. Press and hold Shift and then press these
-    buttons jumps 32 beats. *This section needs further development.*
-    **HELP APPRECIATED**
+43. **< / > Param 1/2:** These are currently mapped to add/remove rating
+    stars to the loaded track. Press and hold Shift and then press these
+    buttons to change track color in the library.
+    *This section needs further development.* **HELP APPRECIATED**
```

Comment:
What does this mean? Do you need help with documentation or with coding the mapping itself? 

If it's the the latter, does the mapping work as described here and just lacking additional features? I'd rather put a separate "Known issues" section at the bottom. If you really want to put this here, you should probably use a [Sphinx note](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html).

Reply:
Thanks! It was meant to get help for the mapping code but I changed the text and mention there are no 2nd and 3rd functions on the PAD buttons.